components/StatusIcon/vue/tsconfig.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": "../../../tsconfig.vue.json",
+  "include": ["*.vue", "*.ts", "*.tsx", "../../../cypress/support/*.ts"]
```

add `../constants.ts`
```suggestion
  "include": ["*.vue", "*.ts", "*.tsx", "../../../cypress/support/*.ts", "../constants.ts"]
```